app/views/mybookings/bookings/new.html.haml

```diff
@@ -22,9 +25,17 @@
 
           .row
             .col-xs-12.col-sm-6
-              = f.input :recurrent_type, collection: recurrent_types_list, as: :select, include_blank: false, hint: t('mybookings.bookings.hints.recurring_hint')
-            .col-xs-12.col-sm-6
-              = f.input :until_date, as: :datetimepicker, hint: t('mybookings.bookings.hints.until_date_hint')
+              = f.input :recurrent_type,
+                  collection: recurrent_types_list,
+                  as: :select,
+                  include_blank: t('mybookings.bookings.recurrent_types.no_repeat'),
+                  selected: '',
+                  hint: t('mybookings.bookings.hints.recurring_hint')
+            .col-xs-12.col-sm-6.js-dynamic-form-element-to-hide
+              = f.input :until_date,
+                  as: :datetimepicker,
+                  hint: t('mybookings.bookings.hints.until_date_hint'),
+                  input_html: { data: { 'dynamic-form-disables-with': '#booking_recurrent_type=' } }
```

Comment:
Does it works with `'#booking_recurrent_type=""'`. If it works, it would be more understandable.

Reply:
Yeah, I tried what you suggested and it didn't work. Your suggestion is actually a text string with two double quotes, not an empty string. I'm gonna leave a comment in the code that explains the reason.